.idea/checkstyle-idea.xml

```diff
@@ -3,15 +3,16 @@
   <component name="CheckStyle-IDEA">
     <option name="configuration">
       <map>
-        <entry key="active-configuration" value="PROJECT_RELATIVE:$PROJECT_DIR$/.idea/codeStyles/checkstyle.xml:CheckStyle" />
-        <entry key="checkstyle-version" value="8.24" />
+        <entry key="active-configuration" value="LOCAL_FILE:D:/Student Data/Documents/checkstyle.xml:check" />
```

Comment:
Having an absolute directory will break checkstyle on all of our machines. Make sure to use relative directories